images/source/cbs-oranges7-test64-common-el7-build.repo

```diff
@@ -0,0 +1,8 @@
+[oranges7-test64-common-el7-build]
+name = CBS Oranges7
+baseurl = http://cbs.centos.org/kojifiles/repos/oranges7-test64-common-el7-build/latest/$basearch
+enabled = 1
+gpgcheck = 0
+sslverify = 0
+skip_if_unavailable = 1
+includepkgs = golang* protobuf*
```

Comment:
@sdodson @tdawson may be able to get these into the appropriate 3.6 repos for the PaaS SIG soon?